ephemeris/shed_tools.py

```diff
@@ -657,7 +694,53 @@ def install_repositories(self):
                 t.get('changeset_revision', "")
             ) for t in self.errored_repositories])
         )
-        log.info("All repositories have been processed.")
+        log.info("All repositories have been installed.")
+        if self.test:
+            installed_tools = []
+            target_repos = self.installed_repositories
+            if self.test_existing:
+                target_repos += self.skipped_repositories
+            for installed_repository in target_repos:
+                repo_tools = tools_for_repository(self.gi, installed_repository)
+                installed_tools.extend(repo_tools)
+
+            test_results = []
+
+            for tool in installed_tools:
+                galaxy_interactor_kwds = {
```

Comment:
This whole code block can be in a `test_tool` method. This makes it a lot cleaner:
```python
for tool in installed_toos:
  self.test_tool(tool)

Reply:
Agreed - I'll try to make this change.